catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/XstreamTreeWriter.java

```diff
@@ -51,7 +53,7 @@ void startVisit(final String node) {
 
             if (value != null) {
 
-                if (currentPath.isAncestor(path) && path.toString()
+                if (isParent(currentPath, path) && path.toString()
```

Comment:
❓ Not sure I understand why `isAncestor()` wasn't working here. We may want to provide an explanation because somebody may be tempted to change it back.